examples/crm/src/tasks/AddTask.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from 'react';
+import { useState } from 'react';
+import ControlPointIcon from '@mui/icons-material/ControlPoint';
+import {
+    Box,
+    Chip,
+    Dialog,
+    Button,
+    DialogTitle,
+    DialogContent,
+    DialogActions,
+    MenuItem,
+    Stack,
+    TextField,
+} from '@mui/material';
+import { RecordRepresentation, useCreate, useRecordContext } from 'react-admin';
+
+const taskTypes = [
+    'None',
+    'Email',
+    'Demo',
+    'Lunch',
+    'Meeting',
+    'Follow-up',
+    'Thank you',
+    'Ship',
+];
+
+export const AddTask = () => {
+    const [create, { isPending }] = useCreate();
+    const contact = useRecordContext();
+    const [open, setOpen] = useState(false);
+    const handleOpen = () => {
+        setOpen(true);
+    };
+    const handleCreateTask = (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        const form = event.currentTarget;
+        const formData = new FormData(form);
+        const formJson = Object.fromEntries(formData.entries());
+        create(
+            'tasks',
+            { data: { ...formJson, contact_id: contact?.id } },
+            {
+                onSuccess: () => {
+                    setOpen(false);
+                },
+                onError: error => console.error(error),
+            }
+        );
+    };
+    return (
+        <>
+            <Box mt={1}>
+                <Chip
+                    icon={<ControlPointIcon />}
+                    size="small"
+                    variant="outlined"
+                    onClick={handleOpen}
+                    label="Add task"
+                    color="primary"
+                />
+            </Box>
+            <Dialog
+                open={open}
+                onClose={() => setOpen(false)}
+                aria-labelledby="form-dialog-title"
+                fullWidth
+                maxWidth="sm"
+            >
+                <form onSubmit={handleCreateTask}>
```

Comment:
Why don't you use a `Form` here and our validation? And why not `CreateBase` instead of the hook?

Reply:
Seemed overkill. What more would it bring?

Reply:
Consistency regarding how errors are displayed in the app?

Reply:
We're using the browser native validation, so we're consistent with other sites. Thats Jakob's Law of UX ;)

https://lawsofux.com/jakobs-law/

Reply:
Don't we use react-admin validation for the other forms in this app ?

Reply:
You won, I fixed the code